docs/release/trg-0/kit-artifacts.md

```diff
@@ -0,0 +1,255 @@
+
+### Logic / Schema
+
+Mandatory for every KIT that is related to a use case + for the 'incubating' stage
+
+Logic / schemas provides a definition of the minimum viable business process that must be implemented to enable interoperability.
+
+Deliverables: Description
+
+### Business Process
+
+Mandatory for every KIT that is related to a use case + for the 'incubating' stage
+
+Data sovereignty is one of the main principles of Catena-X and a major topic for the participants in the data space. This part should help to understand how the business process works and provides a blue print on how access & usage policies **could** look like.
+
+- Business Architecture
+- Access & Usage Policies
+
+### Standards
+
+Mandatory for every KIT that wants to go to the 'incubating' stage
+
+Links to the relevant standards (Association Library). Add the relevant number of the standard, the name and version of the standard.
+
+`Example:`
+CX - 001 Standardname (Version 2.0)
+
+### Notice
+
+Mandatory for every KIT and all graduation stages
+
+KIT documentation works under the CC-BY-4.0 license. Therefore you need to add the "notice" part to make transparent which companies worked on this KIT.
+
+`Example:`
+
+This work is licensed under the [CC-BY-4.0](https://creativecommons.org/licenses/by/4.0/legalcode).
+
+- SPDX-License-Identifier: CC-BY-4.0
+- SPDX-FileCopyrightText: 2023,2023 ZF Friedrichshafen AG
+- SPDX-FileCopyrightText: 2023,2023 Bayerische Motoren Werke Aktiengesellschaft (BMW AG)
+- SPDX-FileCopyrightText: 2023,2023 SAP SE
+- SPDX-FileCopyrightText: 2023,2023 Volkswagen AG
+- SPDX-FileCopyrightText: 2023,2023 Robert Bosch GmbH
+- SPDX-FileCopyrightText: 2023,2023 Mercedes Benz Group
+- SPDX-FileCopyrightText: 2023,2023 BASF SE
+- SPDX-FileCopyrightText: 2023,2023 Schaeffler AG
+- SPDX-FileCopyrightText: 2023,2023 Contributors to the Eclipse Foundation
+- Source URL: [BPDM GitHub repository](https://github.com/eclipse-tractusx/bpdm)
+
+## Development View
+
+The development view provides developers with resources to accelerate the development of apps and services.
+
+### API Specifications
+
+Mandatory for every KIT that wants to go to the 'incubating' stage
+
+Deliverables:
+
+- Introduction to the API / specification of the API
+- Whats the main focus of the API
+- Integration of the OpenAPI file
+- Use case examples for the API endpoints and how to use them
+
+`Example from DataChain KIT:` The API of the Item Relationship Service (IRS) for retrieving item graphs along the value chain of Catena-X partners.
+
+[`Example for OpenAPI integration:`](/docs-kits/kits/Business%20Partner%20Kit/Software%20Development%20View/Pool%20Api/business-partner-data-management-pool)
+
+### Sample Data
+
+Deliverable:
+
+- Description on how to use the sample data
+- File with sample data
+
+### Reference Implementation
```

Comment:
Reference implementations are a deliverable, not a part of a documentation or description of a development. I further don't get the difference between "Operation View" and "Reference Implementation", since the operation view describes how reference implementations are deployed.